### ISSUE TITLE
[BUG] Markdown plugin doesn't render wiki links inside of Markdown

### ISSUE THREAD

**Issue by bmann** [Jermolene/TiddlyWiki5#6045]
Even with renderWikiText true and seemingly the correct renderWikiTextPragma, `[[Wiki Links]]` do not get rendered as links inside of Markdown tiddlers.

If CamelCase is turned _on_, CamelCase links ARE rendered.

This is reproducible on the [MarkdownDemo site](https://tiddlywiki.com/plugins/tiddlywiki/markdown/):

<img width="1137" alt="Screen Shot 2021-09-17 at 1 36 55 PM" src="https://user-images.githubusercontent.com/280420/133850669-c9a00355-fdfd-4402-a122-c1b4fe4e6d15.png">

See Talk TW thread for extended discussion https://talk.tiddlywiki.org/t/markdown-plugin-rendering-wikitext-links/772

**Comment by saqimtiaz** [comment]
Try this but I will admit it has been quite some time since I looked at the relevant code.

* Edit the config tiddler for the markdown tiddler for the renderWikiTextPragma
* Make sure it includes `prettylink`
* Save and reload the wiki

There is an issue somewhere where the need for a configuration UX for the pragma for markdown was discussed.

**Comment by saqimtiaz** [comment]
wikilink pragma = camel case
prettylink = [[title]] or [[label|title]]

**Comment by linonetwo** [comment]
```
\rules only html image macrocallinline syslink transcludeinline wikilink prettylink filteredtranscludeblock macrocallblock transcludeblock 
```

this is not working
<img width="443" alt="截屏2021-11-04 下午1 37 57" src="https://user-images.githubusercontent.com/3746270/140263480-020a3167-7499-48c5-bff1-bedaf60e9be7.png">

**Comment by linonetwo** [comment]
Oh ,it needs a reload.

I think this should be the default...

**Comment by transce080** [comment]
I also agree this should be the default.  This is the expected behavior when renderWikiText is set to true.
